pkg/api/server/v1alpha2/server.go

```diff
@@ -114,3 +150,66 @@ func withGetResultID(f getResultID) Option {
 		s.getResultID = f
 	}
 }
+
+func (s *Server) createLogPluginServer() error {
+	s.LogPluginServer = &LogPluginServer{
+		config: s.config,
+		logger: s.logger,
+		auth:   s.auth,
+		db:     s.db,
+	}
+
+	if s.config.LOGS_TYPE != string(v1alpha3.LokiLogType) {
+		return nil
+	}
+
+	s.LogPluginServer.IsLogPluginEnabled = true
+
+	labels := strings.Split(s.config.LOGGING_PLUGIN_STATIC_LABELS, ",")
+	for _, v := range labels {
+		label := strings.Split(v, "=")
+		if len(label) != 2 {
+			return fmt.Errorf("incorrect format for LOGGING_STATIC_LABELS: %s", v)
+		}
+		s.LogPluginServer.staticLabels += label[0] + `="` + label[1] + `",`
+	}
+
+	maxRetention := os.Getenv(maxRetentionKey)
+	v, err := strconv.Atoi(maxRetention)
+	if err != nil {
+		s.logger.Warnf("incorrect configuration for maxRetention: %s: setting to default", err.Error())
```

Comment:
This value is taken from Retention Policy Config. Maybe I should document this also.
If the value is incorrect, I think we shouldn't stop the server. That's why I set the default. Maybe I should put `errorf` instead of `warnf`.

Reply:
IMO we should fail. The worst thing is to lie to the admin user who set that value. If we do not fail and instead of silently change the value, the admin will think everything was ok and the system is running with the intended configuration while in fact it is not. @vdemeester WDYT?

Reply:
We are failing the Retention Policy Agent, so the admin will be aware of this. But now that I think, it's possible someone doesn't monitor Agent but only API and Watcher. So we should fail that also.